### ISSUE TITLE
Suez isn't working

### ISSUE THREAD

**Issue by 19jm81** [pippyn/Home-Assistant-Sensor-Afvalbeheer#172]
Keep getting the message unknown, my config is this:

# Afval sensor
    
  - platform: afvalbeheer
    wastecollector: Suez            
    resources:                     
    - restafval
    - gft
    - papier
    - pmd
    postcode: 6811BW                 
    streetnumber: 5

**Comment by pippyn** [comment]
The indentation is wrong...
It should be something like:
```
  - platform: afvalbeheer
    wastecollector: Suez
    - resources:
    - restafval
    - gft
    - papier
    - pmd
    postcode: 6811BW
    streetnumber: 5
```

**Comment by 19jm81** [comment]
Including this? 
    - resources:
This gives an indentation error

**Comment by pippyn** [comment]
No I'm sorry, like this:
```
  - platform: afvalbeheer
    wastecollector: Suez
    resources:
    - restafval
    - gft
    - papier
    - pmd
    postcode: 6811BW
    streetnumber: 5
```

**Comment by 19jm81** [comment]
I do get this in the logs (got that already before submitting issue):

Could the adresses be invalid? Do I need to add floor number?

Afvalinfo Eusebiusbinnensingel Tomorrow changed to none
8:42:15 AM - 4 minutes ago

But my dashboard:

type: entities
entities:
  - entity: sensor.afvalinfo_eusebiusbinnensingel_gft
  - entity: sensor.afvalinfo_eusebiusbinnensingel_papier
  - entity: sensor.afvalinfo_eusebiusbinnensingel_pbd
  - entity: sensor.afvalinfo_eusebiusbinnensingel_restafval
  - entity: sensor.afvalinfo_eusebiusbinnensingel_textiel
  - entity: sensor.afvalinfo_eusebiusbinnensingel_today
  - entity: sensor.afvalinfo_eusebiusbinnensingel_tomorrow

Keeps saying:

Afvalinfo Eusebiusbinnensingel GFT
Unknown
Afvalinfo Eusebiusbinnensingel Papier
Unknown
Afvalinfo Eusebiusbinnensingel PBD
Unknown
Afvalinfo Eusebiusbinnensingel Restafval
Unknown
Afvalinfo Eusebiusbinnensingel Textiel
Unknown
Afvalinfo Eusebiusbinnensingel Today
none
Afvalinfo Eusebiusbinnensingel Tomorrow
non

**Comment by pippyn** [comment]
You can use a suffix like so:
No I'm sorry, like this:
```
  - platform: afvalbeheer
    wastecollector: Suez
    resources:
    - restafval
    - gft
    - papier
    - pmd
    postcode: 6811BW
    streetnumber: 5
    suffix: A
```

**Comment by pippyn** [comment]
If that doesn't work for some reason, you can also use another streetnumber with the same collection schedule

**Comment by 19jm81** [comment]
Tried that too, but still unknown

**Comment by pippyn** [comment]
What does the log say?

**Comment by liudger** [comment]
I got the same unknown error. But these are only for the dates that are next years. I'll try a manual postman request

**Comment by liudger** [comment]
On my end all the info from the garbage collector is null so it won't be retrieved.

**Comment by raymajor** [comment]
Here Also the same "Unknown" error with Suez for the date's next year.

**Comment by 19jm81** [comment]
I've switched to afvalinfo that works fine so far

**Comment by raymajor** [comment]
The problem at Suez is that there are no dates at the agenda for 2021.
Whe have to wait.....

**Comment by pippyn** [comment]
@19jm81 the issue with address 6811BW nr5 is fixed in v4.7.7
